pmlib/Resonator2D.py

```diff
@@ -0,0 +1,77 @@
+
+from math import sqrt
+
+import numpy as np
+from scipy.sparse import dia_matrix,identity
```

Comment:
```suggestion
from scipy.sparse import identity
```

scipy.sparse.dia_matrix imported, but unused.